specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/ManagedInstanceOperations.json

```diff
@@ -0,0 +1,358 @@
+            "description": "The request for getting managed instance operations has been executed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ManagedInstanceOperationListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List the managed instance management operations": {
+            "$ref": "./examples/ListManagedInstanceOperations.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagedInstanceOperationListResult": {
+      "description": "The response to a list managed instance operations request",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedInstanceOperation"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ManagedInstanceOperationProperties": {
+      "description": "The properties of a managed instance operation.",
+      "type": "object",
+      "properties": {
+        "managedInstanceName": {
+          "description": "The name of the managed instance the operation is being performed on.",
+          "type": "string",
+          "readOnly": true
+        },
+        "operation": {
+          "description": "The name of operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "operationFriendlyName": {
+          "description": "The friendly name of operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "percentComplete": {
+          "format": "int32",
+          "description": "The percentage of the operation completed.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "startTime": {
+          "format": "date-time",
+          "description": "The operation start time.",
+          "type": "string",
+          "readOnly": true
+        },
+        "state": {
+          "description": "The operation state.",
+          "enum": [
+            "Pending",
+            "InProgress",
+            "Succeeded",
+            "Failed",
+            "CancelInProgress",
+            "Cancelled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ManagementOperationState",
+            "modelAsString": true
+          }
+        },
+        "errorCode": {
```

Comment:
is there min and max?

Reply:
I am not sure that I understand what you mean.

Reply:
Also can you add other people for ARM team, I don't have permission to do that. Jared is on vacation until the end of the year.